src/main/scripts/driver_details.yaml

```diff
@@ -1,9 +1,9 @@
 modules:
   postgresql-plugin:
-    url: https://repo1.maven.org/maven2/org/jumpmind/symmetric/jdbc/postgresql/9.2-1002-jdbc4/postgresql-9.2-1002-jdbc4.jar
+    url: https://repo1.maven.org/maven2/org/postgresql/postgresql/42.6.0/postgresql-42.6.0.jar
     driver_prop: '[{"name":"postgresql","type":"jdbc","className":"org.postgresql.Driver","description":"Plugin for the PostgreSQL JDBC driver"}]'
     artifact_name: 'postgresql'
-    artifact_version: '9.2-1002-jdbc4'
+    artifact_version: '42.6.0'
   mssql-plugin:
     url: https://repo1.maven.org/maven2/com/microsoft/sqlserver/mssql-jdbc/8.2.1.jre8/mssql-jdbc-8.2.1.jre8.jar
```

Comment:
not related to this PR, but could you pls also change the version of sql server driver to match the version in hub?